atom/browser/auto_updater_mac.mm

```diff
@@ -39,8 +41,13 @@
   return update_url_;
 }
 
+void AutoUpdater::SetFeedURL(std::string feedURL) {
+  update_url_ = feedURL;
+  AutoUpdater::SetFeedOptions(update_url_, serverType_, headers_);
+}
+
 // static
-void AutoUpdater::SetFeedURL(mate::Arguments* args) {
+void AutoUpdater::Initialize(mate::Arguments* args) {
```

Comment:
If I'm reading this right this usage is not possible.

```js
autoUpdater.feedUrl = 'my.feed.url'
autoUpdater.checkForUpdates()
```

i.e. the call to initialize is required, that's not immediately obvious and I think we should remove the footgun by making both `SetFeedURL` and `Initialize` set up the `g_updater`

Reply:
@MarshallOfSound it is possible.  `AutoUpdater::SetFeedOptions` sets up the `g_updater`.  The only difference between `autoUpdater.feedUrl = 'my.feed.url'` and `autoUpdater.initialize(...)` is that you can pass server type and headers to initialize

Reply:
Ah I see, missed that third method 👍